File: March 3, 2026.js
function countPerfectCubes(a, b) {

  let total = 0;
  let root = 0;
  let min = Math.min(a, b);
  let max = Math.max(a, b);

  for (let i = min; i <= max; i++) {
    root = Math.round(Math.pow(Math.abs(i), 1/3) * Math.sign(i))

    if (root ** 3 == i) {
      total++;
    }
  }

  return total;
}

File: March 3, 2026.py
import math

def count_perfect_cubes(a, b):

    total = 0

    for i in range(min(a, b), max(a, b) + 1):
        root = round(math.copysign(abs(i) ** (1/3), i))

        if root ** 3 == i:
            total += 1
    return total